src/commands/game_commands/play.ts

```diff
@@ -1153,6 +1263,12 @@ export default class PlayCommand implements BaseCommand {
             }
         }
 
+        if (gameType === GameType.CLIP) {
+            if (guildPreference.isGuessTimeoutSet()) {
```

Comment:
Why do we set this to null? Is this used in clip mode?

Reply:
Timer shouldn't be allowed with clip mode, so it's reset, and a precheck was added to prevent modifying /timer when in clip mode

Reply:
If it's not allowed can't we just ignore it?

Reply:
It can be done by ignoring when the timer is fired. However, since this isn't a game option, I won't be able to mark it as a conflicting option, so it may cause confusion on why the timer isn't working when it's set

Reply:
I may be able to just hide the /timer entry in /options, let me try that

Reply:
would try to avoid changing peoples options if possible

Reply:
Since there was already similar logic to ignore /goal in elimination, I followed that pattern and will show a conflict via game type

Reply:
we should fix that eventually